src/FirePokemon.java

```diff
@@ -0,0 +1,40 @@
+package src;
+
+public class FirePokemon extends Pokemon {
+
+    //instance variables
+    private String special;
+
+    //constructor
+    public FirePokemon(String name, int weight, int height, int hp, String sound, String food, String special) {
+        super(name, weight, height, hp, sound, food);
+        this.special = special;
+    }
+
+    @Override
+    public void speaks(String sound) {
```

Comment:
You've implemented the eats() and speaks() method the exact same way in every subclass. If you do that, you might as well implement it in de super class. If you make the methods do something different in every subclass, it makes more sense to have it be an abstract class. For insance, in this class you could make eats() print (name + "burns" + food +"and eats it"). In the waterpokemon class you could make it print (name + "soaks" + food + "in water and eats it")